.github/workflows/post-merge-jobs.yml

```diff
@@ -63,6 +63,8 @@ jobs:
                 cd src/Aquifer.Jobs
                 dotnet build --configuration Release --output ../../build --runtime win-x64
                 cd ../..
+            - name: Bundle migrations
+              run: dotnet ef migrations bundle --startup-project src/Aquifer.Migrations --project src/Aquifer.Data --output build/Migrate --self-contained
```

Comment:
Fun fact I learned yesterday: Thanks to using migration bundles all of the migrations are now run inside a transaction (as of .net 9) and thus they all roll back if a single migration step fails.  See https://github.com/dotnet/efcore/issues/22616#issuecomment-2324933520.